Add link to the accessibility resources call recording

GBKS opened the pull request:
This is the call link being added:
https://github.com/BitcoinDesign/Meta/issues/780

Question for @jason-me, should we also add the link to the website you put together [here](https://jason-me.github.io/bitcoin-universal-design/)? Or any other links?

🫀[Check the preview](https://deploy-preview-1165--bitcoin-design-site.netlify.app/guide/designing-products/accessibility/)🫁

swedishfrenchpress commented:
I don’t think this is related to this PR, but it’s something I noticed while reviewing it:
- I expected to find this under the Resources section.
- When I didn’t find it there, I checked the file changes and saw it’s actually in Designing Bitcoin Products → Accessibility.
- There are a bunch of usability testing resources there. I wonder if we should consider linking those in the Resources page.

swedishfrenchpress commented:
@GBKS changes look good. Simple PR. Should I merge as well?

jason-me commented:
Yes, that would be ideal.

jason-me commented:
Yes, that would be ideal.

> This is the call link being added:
> https://github.com/BitcoinDesign/Meta/issues/780
> 
> Question for @jason-me, should we also add the link to the website you put together [here](https://jason-me.github.io/bitcoin-universal-design/)? Or any other links?
> 
> 🫀[Check the preview](https://deploy-preview-1165--bitcoin-design-site.netlify.app/guide/designing-products/accessibility/)🫁

swedishfrenchpress commented:
> Yes, that would be ideal.
> 
> Here is the link to the guide: https://jason-me.github.io/bitcoin-universal-design/ Here is the link to the accessible version of the slide deck: https://speakerdeck.com/rrbtc/designing-for-the-edges-accessibility-resources-for-designers-and-developers And here is a link to the video with captions: https://www.youtube.com/watch?v=QR8A2jjbYhE
> 
> > This is the call link being added:
> > [BitcoinDesign/Meta#780](https://github.com/BitcoinDesign/Meta/issues/780)
> > Question for @jason-me, should we also add the link to the website you put together [here](https://jason-me.github.io/bitcoin-universal-design/)? Or any other links?
> > 🫀[Check the preview](https://deploy-preview-1165--bitcoin-design-site.netlify.app/guide/designing-products/accessibility/)🫁

I think we should probably open a separate PR if we want to include some of these resources. @GBKS WDYT?

GBKS commented:
Thanks for the input. Made a few changes:
- Added the speaker deck and video links to the YouTube video description
- Added the accessibility guide and the test video to the accessibility page

That placement makes sense to me, but let me know if you'd do things differently.
